src/models/FGAtmosphere.h

```diff
@@ -232,9 +232,9 @@ class FGAtmosphere : public FGModel {
   /// Calculate the atmosphere for the given altitude.
   void Calculate(double altitude);
 
-  virtual double CalculateDensityAltitude(const double altitude) { return altitude; }
+  virtual double CalculateDensityAltitude(double density, double geoalt) { return geoalt; }
 
-  virtual double CalculatePressureAltitude(const double altitude) { return altitude; }
+  virtual double CalculatePressureAltitude(double pressure, double geoalt) { return geoalt; }
```

Comment:
Same as for `CalculateDensityAltitude`:
* `geoalt` not clear
* The `const`-ness of the `double` parameters is dropped ?